examples.template.html

```diff
@@ -0,0 +1,101 @@
+<html>
```

Comment:
I cont think that the root directory is the best place to put all this. Maybe we introduce a "scripts" folder or something!?

Reply:
Done.